AggregateMetrics/AggregateMetrics/AzureWebApp/RateCounter.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using System;
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    /// <summary>
+    /// Struct for metrics dependant on time.
+    /// </summary>
+    public struct RateCounter
```

Comment:
the idea was to make it implement the same interface as "FlexiblePerformanceCounterGauge". This way aggregation module will know how to get the value of a rate counter.